Gemfile

```diff
@@ -6,6 +6,7 @@ gem "dotenv-rails", "~> 2.0.0", require: "dotenv/rails-now"
 
 gem "audited-activerecord", "~> 4.0.0"
 gem "coffee-rails", "~> 4.1.0"
+gem "chronic"
```

Comment:
Do we have to configure chronic to parse dd/dd/dddd as dd/mm/yyyy instead of the US mm/dd/yyyy?

Worth adding a few tests to ensure this works as we want it to.